[18-May-2022 10:58:47 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 10:58:47 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 10:58:58 Europe/Berlin] Uncaught exception: 'Exception' with message 'Controller class App\Controllers\Posts not found'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('posts/add-new')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 94
[18-May-2022 10:58:58 Europe/Berlin] PHP Fatal error:  Uncaught ErrorException: require(App/Views/common/head.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php:2
Stack trace:
#0 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): Core\Error::errorHandler(2, 'require(App/Vie...', 'C:\\xampp\\htdocs...', 2)
#1 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): require()
#2 C:\xampp\htdocs\DynamicWebApplication\Core\View.php(17): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\DynamicWebApplication\Core\Error.php(57): Core\View::render('500.php')
#4 [internal function]: Core\Error::exceptionHandler(Object(Exception))
#5 {main}
  thrown in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php on line 2
[18-May-2022 11:56:16 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 11:56:16 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 11:56:17 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 11:56:17 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 11:57:41 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 11:58:29 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 11:58:30 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 13:10:32 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 13:10:44 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 13:10:53 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 13:11:02 Europe/Berlin] Uncaught exception: 'Error' with message 'Undefined constant "lt"'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\Core\View.php(17): require()
#1 C:\xampp\htdocs\DynamicWebApplication\App\Controllers\Login.php(25): Core\View::render('Login/login.php')
#2 [internal function]: App\Controllers\Login->loginAction(Array)
#3 C:\xampp\htdocs\DynamicWebApplication\Core\Controller.php(20): call_user_func(Array, Array)
#4 C:\xampp\htdocs\DynamicWebApplication\Core\Router.php(86): Core\Controller->__call('login', Array)
#5 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('Login/login')
#6 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\App\Views\Login\login.php' on line 7
[18-May-2022 13:11:02 Europe/Berlin] PHP Fatal error:  Uncaught ErrorException: require(App/Views/common/head.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php:2
Stack trace:
#0 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): Core\Error::errorHandler(2, 'require(App/Vie...', 'C:\\xampp\\htdocs...', 2)
#1 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): require()
#2 C:\xampp\htdocs\DynamicWebApplication\Core\View.php(17): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\DynamicWebApplication\Core\Error.php(57): Core\View::render('500.php')
#4 [internal function]: Core\Error::exceptionHandler(Object(Error))
#5 {main}
  thrown in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php on line 2
[18-May-2022 13:11:02 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 13:12:29 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 13:12:29 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 13:12:30 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 13:12:30 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 13:12:31 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 14:15:09 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 14:15:09 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 14:15:09 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 14:15:10 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 14:19:49 Europe/Berlin] Uncaught exception: 'Error' with message 'Call to undefined method App\Controllers\Test::test()'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('Test/test')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 86
[18-May-2022 14:19:49 Europe/Berlin] PHP Fatal error:  Uncaught ErrorException: require(App/Views/common/head.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php:2
Stack trace:
#0 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): Core\Error::errorHandler(2, 'require(App/Vie...', 'C:\\xampp\\htdocs...', 2)
#1 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): require()
#2 C:\xampp\htdocs\DynamicWebApplication\Core\View.php(17): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\DynamicWebApplication\Core\Error.php(57): Core\View::render('500.php')
#4 [internal function]: Core\Error::exceptionHandler(Object(Error))
#5 {main}
  thrown in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php on line 2
[18-May-2022 16:22:37 Europe/Berlin] Uncaught exception: 'Exception' with message 'Controller class App\Controllers\Test not found'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('Test/test')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 94
[18-May-2022 16:22:37 Europe/Berlin] PHP Fatal error:  Uncaught ErrorException: require(App/Views/common/head.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php:2
Stack trace:
#0 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): Core\Error::errorHandler(2, 'require(App/Vie...', 'C:\\xampp\\htdocs...', 2)
#1 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): require()
#2 C:\xampp\htdocs\DynamicWebApplication\Core\View.php(17): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\DynamicWebApplication\Core\Error.php(57): Core\View::render('500.php')
#4 [internal function]: Core\Error::exceptionHandler(Object(Exception))
#5 {main}
  thrown in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php on line 2
[18-May-2022 16:23:37 Europe/Berlin] Uncaught exception: 'Exception' with message 'Controller class App\Controllers\Test not found'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('Test/test')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 94
[18-May-2022 16:23:41 Europe/Berlin] Uncaught exception: 'Exception' with message 'Controller class App\Controllers\Test not found'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('Test/test')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 94
[18-May-2022 16:23:41 Europe/Berlin] PHP Fatal error:  Uncaught ErrorException: require(App/Views/common/head.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php:2
Stack trace:
#0 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): Core\Error::errorHandler(2, 'require(App/Vie...', 'C:\\xampp\\htdocs...', 2)
#1 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): require()
#2 C:\xampp\htdocs\DynamicWebApplication\Core\View.php(17): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\DynamicWebApplication\Core\Error.php(57): Core\View::render('500.php')
#4 [internal function]: Core\Error::exceptionHandler(Object(Exception))
#5 {main}
  thrown in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php on line 2
[18-May-2022 16:24:20 Europe/Berlin] Uncaught exception: 'Exception' with message 'Controller class App\Controllers\Test not found'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('Test/test')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 94
[18-May-2022 16:24:20 Europe/Berlin] PHP Fatal error:  Uncaught ErrorException: require(App/Views/common/head.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php:2
Stack trace:
#0 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): Core\Error::errorHandler(2, 'require(App/Vie...', 'C:\\xampp\\htdocs...', 2)
#1 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): require()
#2 C:\xampp\htdocs\DynamicWebApplication\Core\View.php(17): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\DynamicWebApplication\Core\Error.php(57): Core\View::render('500.php')
#4 [internal function]: Core\Error::exceptionHandler(Object(Exception))
#5 {main}
  thrown in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php on line 2
[18-May-2022 16:24:52 Europe/Berlin] Uncaught exception: 'Exception' with message 'Controller class App\Controllers\Test not found'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('Test/test')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 94
[18-May-2022 16:24:52 Europe/Berlin] PHP Fatal error:  Uncaught ErrorException: require(App/Views/common/head.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php:2
Stack trace:
#0 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): Core\Error::errorHandler(2, 'require(App/Vie...', 'C:\\xampp\\htdocs...', 2)
#1 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): require()
#2 C:\xampp\htdocs\DynamicWebApplication\Core\View.php(17): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\DynamicWebApplication\Core\Error.php(57): Core\View::render('500.php')
#4 [internal function]: Core\Error::exceptionHandler(Object(Exception))
#5 {main}
  thrown in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php on line 2
[18-May-2022 16:24:53 Europe/Berlin] Uncaught exception: 'Exception' with message 'Controller class App\Controllers\Test not found'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('Test/test')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 94
[18-May-2022 16:24:53 Europe/Berlin] PHP Fatal error:  Uncaught ErrorException: require(App/Views/common/head.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php:2
Stack trace:
#0 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): Core\Error::errorHandler(2, 'require(App/Vie...', 'C:\\xampp\\htdocs...', 2)
#1 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): require()
#2 C:\xampp\htdocs\DynamicWebApplication\Core\View.php(17): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\DynamicWebApplication\Core\Error.php(57): Core\View::render('500.php')
#4 [internal function]: Core\Error::exceptionHandler(Object(Exception))
#5 {main}
  thrown in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php on line 2
[18-May-2022 16:24:53 Europe/Berlin] Uncaught exception: 'Exception' with message 'Controller class App\Controllers\Test not found'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('Test/test')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 94
[18-May-2022 16:24:53 Europe/Berlin] PHP Fatal error:  Uncaught ErrorException: require(App/Views/common/head.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php:2
Stack trace:
#0 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): Core\Error::errorHandler(2, 'require(App/Vie...', 'C:\\xampp\\htdocs...', 2)
#1 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): require()
#2 C:\xampp\htdocs\DynamicWebApplication\Core\View.php(17): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\DynamicWebApplication\Core\Error.php(57): Core\View::render('500.php')
#4 [internal function]: Core\Error::exceptionHandler(Object(Exception))
#5 {main}
  thrown in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php on line 2
[18-May-2022 16:24:53 Europe/Berlin] Uncaught exception: 'Exception' with message 'Controller class App\Controllers\Test not found'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('Test/test')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 94
[18-May-2022 16:24:53 Europe/Berlin] PHP Fatal error:  Uncaught ErrorException: require(App/Views/common/head.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php:2
Stack trace:
#0 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): Core\Error::errorHandler(2, 'require(App/Vie...', 'C:\\xampp\\htdocs...', 2)
#1 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): require()
#2 C:\xampp\htdocs\DynamicWebApplication\Core\View.php(17): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\DynamicWebApplication\Core\Error.php(57): Core\View::render('500.php')
#4 [internal function]: Core\Error::exceptionHandler(Object(Exception))
#5 {main}
  thrown in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php on line 2
[18-May-2022 18:47:37 Europe/Berlin] Uncaught exception: 'Exception' with message 'Controller class App\Controllers\Test not found'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('Test/test')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 94
[18-May-2022 18:47:37 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
[18-May-2022 18:47:54 Europe/Berlin] Uncaught exception: 'Exception' with message 'Controller class App\Controllers\Test not found'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('Test/test')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 94
[18-May-2022 18:47:54 Europe/Berlin] PHP Fatal error:  Uncaught ErrorException: require(App/Views/common/head.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php:2
Stack trace:
#0 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): Core\Error::errorHandler(2, 'require(App/Vie...', 'C:\\xampp\\htdocs...', 2)
#1 C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php(2): require()
#2 C:\xampp\htdocs\DynamicWebApplication\Core\View.php(17): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\DynamicWebApplication\Core\Error.php(57): Core\View::render('500.php')
#4 [internal function]: Core\Error::exceptionHandler(Object(Exception))
#5 {main}
  thrown in C:\xampp\htdocs\DynamicWebApplication\App\Views\500.php on line 2
[18-May-2022 18:47:58 Europe/Berlin] Uncaught exception: 'Exception' with message 'Controller class App\Controllers\Test not found'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('Test/test')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 94
[18-May-2022 18:47:58 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched'
Stack trace: #0 C:\xampp\htdocs\DynamicWebApplication\public\index.php(56): Core\Router->dispatch('public/css/styl...')
#1 {main}
Thrown in 'C:\xampp\htdocs\DynamicWebApplication\Core\Router.php' on line 98
